clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -54,6 +58,17 @@ object ClioWebClient {
       extends RuntimeException(
         s"Got an error from the Clio server. Status code: $statusCode. Entity: $entityBody"
       )
+
+  def jsonWithDocumentStatus(input: Json): Json = {
+    import ModelAutoDerivation.encodeEnum
+    input.mapObject(
+      obj =>
+        obj.add(
```

Comment:
This will overwrite any existing document status (which I think isn't the current behavior). I think the old way would be something like `Json.obj("status" -> status).deepMerge(obj)`

Reply:
Where this function is used, it is to set DocumentStatus to normal for queries. Nothing gets overwritten. The old way was to operate on Input classes and call .withDocumentStatus, which uses copy

Reply:
by nothing I mean nothing in Clio. I guess the query could be overwritten if someone is explicitly querying Doc Status not normal, but you can use queryAll for that